book/src/integrations/oauth2.md

```diff
@@ -483,3 +483,65 @@ The `email` scope needs to be passed and thus the mail attribute needs to exist
 ```bash
 kanidm person update <ID> --mail "YYYY@somedomain.com" --name idm_admin
 ```
+
+### Grafana
+
+Grafana is a open source analytics and interactive visualization web application. It provides charts, graphs, and alerts when connected to supported data source.
+
+Prepare the environment:
+
+```bash
+$ kanidm system oauth2 create grafana "grafana.domain.name" https://grafana.domain.name
+$ kanidm system oauth2 update-scope-map grafana grafana_users email openid profile
```

Comment:
Shouldn't this include the groups scope, as it gets requested by Grafana with the config below?

Reply:
Yep, that'd make sense.